### PR TITLE
[EOS-14427] masks secrets in logs

### PR DESCRIPTION
**Pull request by andkononykhin2**

### CODE REVIEW

**Comment by andkononykhin2** [comment]
@ypise @mazinamdar when you will be reviewing the PR please don't wonder changes in test_errors.py : I applied `dos2unix` to that file since it had been saved in windows editor by some reason. That led to some inconvenience.

File: api/python/provisioner/salt.py [hunk 1]
```diff
@@ -798,6 +809,7 @@ def _salt_runner_cmd(  # noqa: C901 FIXME
     fun_args: Union[Tuple, None] = None,
     fun_kwargs: Union[Dict, None] = None,
     nowait=False,
+    secure=False,
```
Comment (ypise) [inline api/python/provisioner/salt.py:812]: Should the default value of secure not be True?

File: api/python/provisioner/salt.py [hunk 2]
```diff
@@ -146,9 +151,11 @@ class SaltRunnerArgs(SaltArgsMixin):
     )
     nowait: bool = False
     kw: Dict = attr.Factory(dict)
+    secure: bool = False
```
Comment (ypise) [inline api/python/provisioner/salt.py:154]: Should the default value of secure not be True?

**Review by ypise — APPROVED** [review]

File: api/python/provisioner/salt.py (hunk 1 above)
Comment (andkononykhin2) [reply api/python/provisioner/salt.py:812]: No, it would impact many places where we don't need that.

File: api/python/provisioner/salt.py (hunk 2 above)
Comment (andkononykhin2) [reply api/python/provisioner/salt.py:154]: No, it would impact many places where we don't need that.

**Pull request merged**